src/pkg/packager/remove.go

```diff
@@ -107,17 +109,49 @@ func (p *Packager) Remove(ctx context.Context) (err error) {
 func (p *Packager) updatePackageSecret(ctx context.Context, deployedPackage types.DeployedPackage) {
 	// Only attempt to update the package secret if we are actually connected to a cluster
 	if p.cluster != nil {
+		// TODO: Error needs to be checked
+		newPackageSecretData, _ := json.Marshal(deployedPackage)
```

Comment:
Not sure if you meant to leave this here, but I think we should handle this error and remove the TODO comment